### PR TITLE
Bug code refactoring from LFA feedback 163131151

### PR DESCRIPTION
**Pull request by Oluwaseyi000**
**What do this PR do?**
This PR refactors the code, fixed errors and code dupplication as recommended by my LFA in https://github.com/Oluwaseyi000/Questioner_Andela/pull/24#partial-pull-merging

**Description of Tasks to be Completed**

- [ ] Remove all unnessary double line spacing

- [ ] Return appropriate status code with each api response

- [ ] Delete duplicate files and folder

- [ ] Remove the compiled src folder from upstream

- [ ] Removd decapritated funcion and replace with  newer version

**How This Should be Mannually Tested**
Compare this current branch with the recommended corrections in the branch https://github.com/Oluwaseyi000/Questioner_Andela/pull/24#partial-pull-merging

**What are the Relevant Pivotal Tracker Stories**
[#163131151](https://www.pivotaltracker.com/n/projects/2232311/#163131151)

### CODE REVIEW

File: server/controller/Question.js [hunk 1]
```diff
@@ -83,7 +81,7 @@ class questionController {
                })
             } else if (req.body.voteType === 'downvote') {
                question.votes = question.votes - 1;
-               return res.json({
+               return res.status(200).json({
                   status: 200,
                   message: `Question upvoted`,
```
Comment (larrystone) [inline server/controller/Question.js]: This should be downvoted and not upvoted. Please update it

File: server/server.js [hunk 2]
```diff
@@ -2,16 +2,12 @@ import express from 'express';
 import router from './routes';
 import bodyParser from 'body-parser'
 
-
 const app = express();
 
-
 app.use(express.json());   
 app.use(bodyParser.json());                     
 app.use(bodyParser.urlencoded({extended: true}));
```
Comment (larrystone) [inline server/server.js:7]: What is the difference between this line and the previous one? Think of what can be done to consolidate them.

**Review by larrystone — CHANGES_REQUESTED** [review]
Nice work, please look into the the concerns below

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,4 @@
 node_modules/
 coverage/
+.nyc_output/
+src/
```
Comment (larrystone) [inline .gitignore]: Why gitignore this when it doesn't do anything in the codebase. What if you delete this folder?

File: server/controller/Meetup.js [hunk 4]
```diff
@@ -139,22 +103,19 @@ class Meetup {
...
                error: `Request unsuccessful, meetup for id ${req.params.meetupId} not found`
             })
-
          } else {
             Meetups.splice(meetup, 1);
-            return res.json({
-               status: 204,
+            return res.status(200).json({
+               status: 200,
                message: `Delete Successful. Meetup with id ${req.params.meetupId} successfully deleted`
```
Comment (larrystone) [inline server/controller/Meetup.js:114]: consider destructuring variables especially in cases where you have to use those variables again. e.g `meetupId` above

**Review by larrystone — CHANGES_REQUESTED** [review]
And these ones too

File: server/controller/Question.js (hunk 1 above)
Comment (Oluwaseyi000) [reply server/controller/Question.js]: Roslved

File: server/server.js (hunk 2 above)
Comment (Oluwaseyi000) [reply server/server.js:7]: i have removed express.json()

File: .gitignore (hunk 3 above)
Comment (Oluwaseyi000) [reply .gitignore]: i've deleted the unnecessary folder

**Pull request merged**